data/themes/default/style.css

```diff
@@ -165,6 +165,17 @@ PianoRoll {
 	qproperty-ghostNoteBorders: false;
 	qproperty-barColor: #078f3a;
 	qproperty-markedSemitoneColor: rgba(255, 255, 255, 30);
+	/* Piano keys */
```

Comment:
I would like to note these updates are purely for testing and by no means are to be considered final.